src/pthreads.c

```diff
@@ -74,7 +74,7 @@
 #endif
 
 /* Alloca */
-#if defined(_MSC_VER)
+#if defined(_MSC_VER) || defined(__MINGW32__)
```

Comment:
Should this be
```suggestion
#if defined(_WIN32)
```
if the relevant check is "the target is Win32", rather than "the compiler is Microsoft Visual Code"?